### ISSUE TITLE
new to electrum version 4.1.5 - connection issues and missing coins.

### ISSUE THREAD

**Issue by LeoGonza7** [spesmilo/electrum#7531]
Hi.
I’m new to your product and i just recently transfered a small amaount of BTC (0,005) to the Electrum wallet. The transfer went well and i could se the funds in the wallet. Then I didnt log back in for about two days.
When i got back in the Electrum wallet it couldnt connect with any of the servers. I tried to restart several times and nothing worked, i scrolled between server channels manually and it still wouldnt work - even though i could see in the “history” column that i had received 0,005 BTC on Friday the 8th oc October 10, 2021. 
Lastly, I erased the wallet and try to recover it with my “seed” code. 
I logged back in and still no connection, and now of course i cant see the transfer in the “history” column anymore.
What can i do to retrieve the BTC and what can i do to avoid this connections issues from happening again?
I use VPN - is that going to be an issue with my wallet?

Appreciate your help, Leonardo Gonzalez.

**Comment by SomberNight** [comment]
@Liisachan OP did not mention Tor at all.

@LeoGonza7 is this issue still relevant? If you still cannot connect to any server, please upload [debug logs](https://electrum.readthedocs.io/en/latest/faq.html#how-to-enable-debug-logging).

**Comment by Liisachan** [comment]
@SomberNight Right, my comment is probably irrelevant. Please disregard it.

However, OP is asking, "It seems that the wallet doesnt work properly when VPN is on, is there any way around that issue?" and one possible solution would be connecting via Tor, assuming that their ISP or VPN provider is somehow blocking Electrum. There seem to be a few anti-crypto countries and Electrum is one of the most popular BTC wallets, so blocking it specifically is not unthinkable (which is not an issue of Electrum, though).

Incidentally, https://bisq.community/ is still using v2 onion, http://dnkeu7zlmu7vlyox.onion/ as of today (Jan 19, 2022), which is no longer accessible with the current version of Tor or Tor Browser, and apparently no one is complaining... I guess Tor is not so commonly used as I thought.

Anyway, thanks again to everyone who is maintaining this awesome project!

**Comment by LeoGonza7** [comment]
Yes, i still have the same issue of not being able to connect to my wallet when VPN is on. Besides that the wallet works fine.

**Comment by SomberNight** [comment]
What kind of VPN are you using? Is it a public commercial product (if so, maybe share which company)?
WireGuard or OpenVPN or sth else?

but most importantly, could you please upload debug logs?
